src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
I don't think providing a way to override the deprecation status in inherited services is a good thing, I would even not allow that at all. Having just one `deprecated` entry sounds better to me in all supported formats.